interactions/api/models/guild.py

```diff
@@ -1905,6 +1905,65 @@ async def get_all_bans(self) -> List[Dict[str, User]]:
 
         return res
 
+    async def prune(
+        self,
+        days: int = 7,
+        compute_prune_count: bool = True,
+        include_roles: Optional[Union[List[Role], List[int]]] = MISSING,
+    ) -> Optional[int]:
+        """
+        Begins a prune operation.
+
+        :param days: Number of days to count. Defaults to 7.
```

Comment:
Give options (min max or smth)

Reply:
Done

Reply:
This is still missing from the helper methods. Most users will look at those and not the http methods

Reply:
I completely forgot. You are 100% right.

Reply:
Pushed!